### PR TITLE
Summon eip-666.md

### PR DESCRIPTION
**Pull request by gcolvin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-666.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+eip: 666
+title: The Number of the Beast
+description: Two horns.
+author: Beelzebub (@בַּע-זְבוּב)
+discussions-to: https://hell.com
+status: Undead
```
Comment (Pandapip1) [inline EIPS/eip-666.md:7]: Doesn't this imply that this was once a living standard?

File: EIPS/eip-666.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## Specification
+
+> [Here is wisdom. Let him that hath understanding count the number of the beast: for it is the number of a man; and his number is Six hundred threescore and six.](https://www.kingjamesbibleonline.org/Revelation-13-8/)
+
+*666* is the Decimal Number of the Beast.  Therefore, the Hexadecimal Address of the Beast is
+* *0x000000000000000000000000000000000000029A*
+
+## Rationale
+
+John of Patmos Revealed the Number of the Beast in AD 95, which Number was Assigned since Genesis.
```
Comment (Pandapip1) [inline EIPS/eip-666.md:35]: How was this verified? Was the hash of the number already known? This seems to have some provability issues.

File: EIPS/eip-666.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+eip: 666
+title: The Number of the Beast
+description: Two horns.
+author: Beelzebub (@בַּע-זְבוּב)
```
Comment (Pandapip1) [inline EIPS/eip-666.md:5]: This Github user doesn't exist.

File: EIPS/eip-666.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+discussions-to: https://hell.com
```
Comment (Pandapip1) [inline EIPS/eip-666.md:6]: Should be a thread on Ethereum theologists

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
There are quite a few issues here.

File: EIPS/eip-666.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+Eternal.
+
+## Reference Implementation
+
+> ```SELFDESTRUCT```
+
+## Security Considerations
+
+> [Lasciate ogne speranza, voi ch'intrate.](http://www.worldofdante.org/comedy/dante/inferno.xml/1.3)
```
[the remark this answers is not in the document]
Comment (gcolvin) [reply EIPS/eip-666.md:47]: This is a normative link.  Follow it at your peril.

File: EIPS/eip-666.md (hunk 3 above)
Comment (drortirosh) [reply EIPS/eip-666.md:5]: It should be בַּעַל זְבוּב

File: EIPS/eip-666.md (hunk 1 above)
Comment (gcolvin) [reply EIPS/eip-666.md:7]: At the limit in infinity, yes.

File: EIPS/eip-666.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+---
+eip: 666
+title: The Number of the Beast
+description: Two horns.
+author: Beelzebub (@בַּע-זְבוּב)
+discussions-to: https://hell.com
+status: Undead
+type: Core
+deceased: 2022-04-01
```
[the remark this answers is not in the document]
Comment (gcolvin) [reply EIPS/eip-666.md:9]: Yes.  I think _Resurrected_ would be correct, but it's hard to tell with the Undead.

File: EIPS/eip-666.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+---
+
+## Abstract
+
+The Address of the Beast is
+* _0x000000000000000000000000000000000000029A_
+
+## Motivation
+
+We propose to assign a pre-defiled Ethereum address to The Beast.
```
[the remark this answers is not in the document]
Comment (gcolvin) [reply EIPS/eip-666.md:19]: The contract at that address will be pre-defiled.

**Review by DONALDSULLENDER — APPROVED** [review]

**Comment by MicahZoltu** [comment]
I'm closing all PRs that were open on April 1st since we received a number of "joke" EIPs.  If this was in fact a serious EIP that you would like to have reviewed, please resubmit.

Note: Please keep in mind that EIP editors are all volunteers and have more work than time available.

**Comment by Pandapip1** [comment]
@gcolvin mind fixing this? The branch appears to have somehow disappeared.

**Pull request closed without merging**